provisioning/service/tests/Config/X509CertificateWithInfoTests.cs

```diff
@@ -95,75 +62,54 @@ public void X509CertificateWithInfoConstructorThrowsOnNullX509Certificate()
                 "-----END CERTIFICATE-----\n";
 
             // act - assert
-#pragma warning disable CA1806 // Do not ignore method results
-            TestAssert.Throws<ArgumentException>(() => new X509CertificateWithInfo(certificateNull));
-            TestAssert.Throws<ArgumentException>(() => new X509CertificateWithInfo(certificateString));
-
-            Action act1 = () => new X509CertificateWithInfo(certificateEmpty);
-            var error1 = act1.Should().Throw<ProvisioningServiceException>();
-            error1.And.StatusCode.Should().Be(HttpStatusCode.BadRequest);
-            error1.And.IsTransient.Should().BeFalse();
-
-            Action act2 = () => new X509CertificateWithInfo(certificateStringEmpty);
-            var error2 = act2.Should().Throw<ProvisioningServiceException>();
-            error2.And.StatusCode.Should().Be(HttpStatusCode.BadRequest);
-            error2.And.IsTransient.Should().BeFalse();
-
-            Action act3 = () => new X509CertificateWithInfo(certificateStringInvalid);
-            var error3 = act3.Should().Throw<ProvisioningServiceException>();
-            error3.And.StatusCode.Should().Be(HttpStatusCode.BadRequest);
-            error3.And.IsTransient.Should().BeFalse();
-#pragma warning restore CA1806 // Do not ignore method results
+            Action act = () => _ = new X509CertificateWithInfo(certificateNull);
```

Comment:
I updated these tests to use FluentAssertions, so I could delete the TestAssert.cs file.